packages/harmony/src/components/button/FilterButton/FilterButton.tsx

```diff
@@ -147,12 +147,13 @@ export const FilterButton = forwardRef(function FilterButton<
   }
 
   const iconCss = size === 'small' ? smallIconStyles : defaultIconStyles
-
   const handleClick = useCallback(() => {
     if (onClick) {
       onClick()
     } else {
-      setIsOpen((isOpen: boolean) => !isOpen)
+      setIsOpen((prev) => {
```

Comment:
i think isOpen is a conflicting name ^

Reply:
Weird you should be able to call it whatever you want

Reply:
yeaa just confusing to reuse the same variable name i think